tutorial/creating_packages/create_your_first_package.rst

```diff
@@ -269,51 +269,51 @@ We can see them with:
 .. code-block:: bash
 
     # list all the binaries built for the hello/1.0 package in the cache
-    $ conan list hello/1.0:*
-    Local Cache:
+    $ conan list "hello/1.0:*"
+    Local Cache
       hello
         hello/1.0
           revisions
-            dcbfe21e5250264b26595d151796be70 (2024-03-04 17:52:39 UTC)
+            dcbfe21e5250264b26595d151796be70 (2024-05-10 09:40:15 UTC)
               packages
-                6679492451b5d0750f14f9024fdbf84e19d2941b
+                2505f7ebb5a4cca156b2d6b8534f415a4a48b5c9
                   info
                     settings
-                      arch: x86_64
+                      arch: armv8
```

Comment:
This changes are confusing to the user if they have been following the tutorial, as we have been showing them x86 output (Line 219 for example)

You might want to update those eariler outputs too, or stick with the x86 compilations

Reply:
Yes... I see, maybe I'll update previous commands in order to show the world we are up to date with mac silicon standards

Reply:
I've updated the 219 lines and other in order to be consistent with arm